bftengine/src/bftengine/ReplicaLoader.cpp

```diff
@@ -53,7 +53,9 @@ ReplicaLoader::ErrorCode loadConfig(LoadedReplicaData &ld) {
                                    config.replicaPrivateKey,
                                    config.publicKeysOfReplicas,
                                    concord::util::crypto::KeyFormat::HexaDecimalStrippedFormat,
-                                   config.clientTransactionSigningEnabled ? &config.publicKeysOfClients : nullptr,
+                                   (config.clientTransactionSigningEnabled || !config.clientsKeysPrefix.empty())
```

Comment:
nit: same logic repeats 3 times.
For better readibility, and also because there is more logic here, maybe we should have an inline function as part of ReplicaConfig?

Reply:
fixed